packages/integration-sdk-cli/src/commands/run.ts

```diff
@@ -13,11 +13,14 @@ import {
   getApiBaseUrl,
   getApiKeyFromEnvironment,
   initiateSynchronization,
-  uploadCollectedData,
+  // uploadCollectedData
```

Comment:
I'm sure you'll do this in a future PR but this should be removed, not just commented out.